cert-manager/Makefile

```diff
@@ -0,0 +1,74 @@
+.PHONY: run
+run: run-kubernetes
+
+kubeconfig.yaml:
+	kind create cluster --name cert-manager-test --kubeconfig ./kubeconfig.yaml
+	helm install --kubeconfig kubeconfig.yaml prometheus prometheus-community/prometheus -f ../.github/configs/prometheus.yaml -n prometheus --create-namespace --wait
+	helm install --kubeconfig kubeconfig.yaml loki grafana/loki -f ../.github/configs/loki.yaml -n loki --create-namespace --wait
+
+.PHONY: run-kubernetes
+run-kubernetes: values.yaml kubeconfig.yaml
+	@helm repo add jetstack https://charts.jetstack.io
+	@helm repo add grafana https://grafana.github.io/helm-charts
+	@helm repo update
+	helm install cert-manager jetstack/cert-manager -n cert-manager --create-namespace --set installCRDs=true --wait --kubeconfig ./kubeconfig.yaml
+	helm install k8s-monitoring grafana/k8s-monitoring -n monitoring --create-namespace --wait --kubeconfig ./kubeconfig.yaml \
+		--values values.yaml \
+		--set-file extraConfig=cert-manager.river \
+		--values test-queries.yaml
+
+.PHONY: test-kubernetes
+test-kubernetes:
+	helm test k8s-monitoring -n monitoring --kubeconfig ./kubeconfig.yaml --logs
```

Comment:
Minor nit, but probably not needed for most runs?

Maybe something we can add for PR merge, but not commits?

Reply:
In fact, this is the part that actually runs the metrics check. Without this, it just deploys things without asserting the success of gathering and saving cert manager metrics.

This section in the action workflow:
```
on:
  pull_request:
    paths:
      - 'cert-manager/**'
```
... means that the workflow only runs on PRs in the cert-manager directory and not on commits.